config/jobs/kubernetes-sigs/azurefile-csi-driver/azurefile-csi-driver-config.yaml

```diff
@@ -0,0 +1,151 @@
+        - --
+        - make
+        - sanity-test
+        securityContext:
+          privileged: true
+    annotations:
+      testgrid-dashboards: sig-azure-master
+      testgrid-tab-name: pr-azurefile-csi-driver-sanity
+      description: "Run sanity tests for Azure File CSI driver."
+      testgrid-num-columns-recent: '30'
+  - name: pull-azurefile-csi-driver-integration
+    always_run: true
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+      preset-cloudprovider-azure-cred: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190920-5a67b1f-master
+        args:
+        - "--job=$(JOB_NAME)"
+        - "--root=/go/src"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--repo=github.com/kubernetes-sigs/azurefile-csi-driver=$(PULL_REFS)"
+        - "--upload=gs://kubernetes-upstream/pr-logs"
+        - "--scenario=execute"
+        - --
+        - make
+        - integration-test
+        securityContext:
+          privileged: true
+    annotations:
+      testgrid-dashboards: sig-azure-master
+      testgrid-tab-name: pr-azurefile-csi-driver-integration
+      description: "Run integration tests for Azure File CSI driver."
+      testgrid-num-columns-recent: '30'
+  - name: pull-azurefile-csi-driver-e2e
+    always_run: true
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+      preset-cloudprovider-azure-cred: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190920-5a67b1f-master
+        args:
+        - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/kubernetes"
+        - "--repo=github.com/kubernetes-sigs/azurefile-csi-driver=$(PULL_REFS)"
+        - "--root=/go/src"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--timeout=450"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--scenario=kubernetes_e2e"
+        - --
+        - "--gce-ssh="
+        - "--test=true"
+        - "--up=true"
+        - "--down=true"
+        - "--deployment=acsengine"
+        - "--build=bazel"
+        - "--provider=skeleton"
+        - "--ginkgo-parallel=1"
+        - "--acsengine-agentpoolcount=2"
+        - "--acsengine-admin-username=azureuser"
+        - "--acsengine-creds=$AZURE_CREDENTIALS"
+        - "--acsengine-orchestratorRelease=1.16"
+        - "--acsengine-mastervmsize=Standard_DS2_v2"
+        - "--acsengine-agentvmsize=Standard_D4s_v3"
+        - "--acsengine-location=eastus2"
+        - "--acsengine-public-key=$AZURE_SSH_PUBLIC_KEY_FILE"
+        - "--acsengine-template-url=https://raw.githubusercontent.com/kubernetes/cloud-provider-azure/master/tests/k8s-azure/manifest/linux.json"
+        - "--acsengine-download-url=https://github.com/Azure/aks-engine/releases/download/v0.39.1/aks-engine-v0.39.1-linux-amd64.tar.gz"
+        - "--test-azure-file-csi-driver=True"
```

Comment:
This `test-azure-file-csi-driver` flag won't take into effect until after a new `gcr.io/k8s-testimages/kubekins-e2e` image has been released with the latest kubetest code. So you might need to create separate PRs for this.

Reply:
Removed the job configuration for now. I will create a separate PR for that after this is merged.